src/app/_components/shared/organisms/NavBar/NavBar.tsx

```diff
@@ -1,14 +1,14 @@
-import { Flex, Link } from '@radix-ui/themes'
+import { Button } from '../../atoms/Button/Button'
 import styles from './NavBar.module.scss'
 
 export const NavBar = () => {
   return (
-    <Flex className={styles['navBar']} asChild p="4">
-      <nav>
-        <Link href="/" underline="none" weight="bold" size="5">
-          Enriconsolas
-        </Link>
-      </nav>
-    </Flex>
+    <nav className={styles['nav-bar']}>
+      <a href="" className={styles['nav-bar__left-actions']}>QEYS</a>
+      <div className={styles['nav-bar__right-actions']}>
+        <Button variant="transparent">Sign In</Button>
```

Comment:
Remember we have the translations system set up. Try to not hardcode text in the app.